third-party-podspecs/react-native-blur.podspec.json

```diff
@@ -1,21 +1,23 @@
 {
-    "name": "react-native-blur",
-    "version": "3.3.0",
-    "summary": "Component implementation for UIVisualEffectView's blur and vibrancy effect.",
-    "description": "Component implementation for UIVisualEffectView's blur and vibrancy effect.",
-    "license": "MIT",
-    "authors": "react-native-community",
-    "homepage": "https://github.com/react-native-community/react-native-blur",
-    "source": {
-        "git": "https://github.com/react-native-community/react-native-blur",
-        "tag": "v3.3.0"
-    },
-    "source_files": "ios/*.{h,m}",
-    "preserve_paths": "**/*.js",
-    "platforms": {
-        "ios": "8.0"
-    },
-    "dependencies": {
-        "React": []
-    }
-}
\ No newline at end of file
+  "name": "react-native-blur",
+  "version": "0.8.0",
```

Comment:
This is strange 😕 We probably won't want this kind of downgrade. I'll check the diff of native iOS files and `react-native-blur`'s podspec.

Reply:
This seems to be wrong in the Podspec, we'll need to fork this repo:
https://github.com/Kureev/react-native-blur/blob/v3.3.0/react-native-blur.podspec#L3

Reply:
@hypest I've created the fork and pushed the related fix: https://github.com/wordpress-mobile/react-native-blur/pull/1/, but I may need some help with setting up Jitpack :(

After it's set up, will push a `v3.6.1-wp` tag in `wp-fork` branch to get it used in the generated podspecs.

Reply:
Thanks for the ping Ceyhun! We had the chance to talk about this (the new fork) during a 1:1 and we said that it's fine for us to not depend on the fork for the initial merge of the RN upgrade, and wrangle it afterwards as part of cleaning up all the fork upgrade PRs.

Reply:
Just added a workaround for now in `generate-podspecs.sh` for react-native-blur specifically and also some warning messages to catch this and also to not modify podspecs manually.